.clang-format

```diff
@@ -0,0 +1,36 @@
+BasedOnStyle: Google
+IndentWidth: 4
+TabWidth: 4
+UseTab: Never
+ColumnLimit: 120
+BreakBeforeBraces: Custom
+BraceWrapping:
+  AfterClass: true
+  AfterControlStatement: true
+  AfterEnum: true
+  AfterFunction: true
+  AfterNamespace: true
+  AfterObjCDeclaration: true
+  AfterStruct: true
+  AfterUnion: true
+  BeforeCatch: true
+  BeforeElse: true
+  IndentBraces: false
+AllowShortIfStatementsOnASingleLine: true
```

Comment:
Do we have such statements?

Reply:
Only in a few places. Basically, the single-line functions look like regular ones. I added it for compactness. I can remove for the sake of consistency.

Reply:
As this PR does not enforces the code style, I think it's fine to keep as is.